components/home/Tools.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import ScaffoldContainer from '../ScaffoldContainer';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    '& .MuiCardContent-root': {
+      padding: '0.5rem 1rem 2rem',
+    },
+    '& .MuiGrid-spacing-xs-10': {
+      padding: '1rem',
+    },
+    paddingLeft: theme.spacing(5),
+    paddingRight: theme.spacing(5),
+  },
+  heading: {
+    fontWeight: 'bold',
+    color: '#598fe0',
+    width: '90%',
+  },
+  content: {
+    fontWeight: 'lighter',
+    color: '#598fe0',
+  },
+  contentContainer: {
+    textAlign: 'left',
+    marginTop: '3rem',
+  },
+  media: {
+    borderRadius: '50%',
+    height: 50,
+    width: 50,
+    marginLeft: '1rem',
+  },
+}));
+
+function Tools(props) {
```

Comment:
I would go ahead and pull the tools content into this file, and rename it to TOOLS_CONTENT (it's a constant). There's no need to pass it as a prop because it will not be changing